mbed-events/equeue/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2016 Christopher Haster
```

Comment:
This comes from mbed-events being built on the little [equeue](https://github.com/geky/equeue) library that I put together outside of mbed. I'm not opposed to Apache 2.0 if the upstream project is unimpacted.